util/docker/docker.go

```diff
@@ -0,0 +1,72 @@
+package docker
+
+import (
+	"net"
+	"time"
+
+	"github.com/bblfsh/performance/util"
+	"github.com/ory/dockertest"
+	"github.com/ory/dockertest/docker"
+	"gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// TODO maybe we need to run container in stateless mode and install driver version that we want
+const (
+	image = "bblfsh/bblfshd"
+	tag   = "latest-drivers"
```

Comment:
We most probably would need to have this as a configurable parameter

Reply:
Done